pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/utils/DriverUtils.java

```diff
@@ -217,4 +220,14 @@ public static boolean queryContainsLimitStatement(String query) {
     Matcher matcher = pattern.matcher(query);
     return matcher.find();
   }
+
+  public static String checkEnableNullHandling(PinotConnection connection, String query) {
```

Comment:
Suggest renaming to `enableNullHandling` because it will modify the query

Reply:
Renamed.